Erring after upgrading to latest version

**Describe the bug**
I upgrade to the latest version 8.0.0-rc.2 and now i am getting this error message 

"node:internal/modules/cjs/loader:960
  const err = new Error(`Cannot find module '${request}'`);
              ^

Error: Cannot find module '/Users/myName/Projects/myProjectName/node_modules/@cucumber/cucumber/lib/formatter/helpers.js'2 "


**Expected behavior**
The code should work as usual before the upgrade


**Desktop (please complete the following information):**
 - OS: macOS Big Sur v 11.6.2
 - Browser chrome
 - Version 96.0.4664.55
- Node Version 16.13.0

**Additional context**
The only change i made was the upgrade to the latest so not sure why i am getting this error

My issue will be that my report depends on it as you can see from that piece of code.
Let me comment it out of the code do some checking and update this later

This is the entire error message:

node:internal/modules/cjs/loader:960
  const err = new Error(`Cannot find module '${request}'`);
              ^

Error: Cannot find module '/Users/larryg/Projects/workProjects/OAF/node_modules/@cucumber/cucumber/lib/formatter/helpers.js'
    at createEsmNotFoundErr (node:internal/modules/cjs/loader:960:15)
    at finalizeEsmResolution (node:internal/modules/cjs/loader:953:15)
    at resolveExports (node:internal/modules/cjs/loader:482:14)
    at Function.Module._findPath (node:internal/modules/cjs/loader:522:31)
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:919:27)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/larryg/Projects/workProjects/OAF/node_modules/@cucumber/pretty-formatter/lib/src/index.js:4:19)
    at Module._compile (node:internal/modules/cjs/loader:1101:14) {
  code: 'MODULE_NOT_FOUND',
  path: '/Users/larryg/Projects/workProjects/OAF/node_modules/@cucumber/cucumber/package.json',
  [Symbol(originalCallSite)]: [
    CallSite {}, CallSite {},
    CallSite {}, CallSite {},
    CallSite {}, CallSite {},
    CallSite {}, CallSite {},
    CallSite {}, CallSite {}
  ],
  [Symbol(mutatedCallSite)]: [
    CallSite {}, CallSite {},
    CallSite {}, CallSite {},
    CallSite {}, CallSite {},
    CallSite {}, CallSite {},
    CallSite {}, CallSite {}
  ]
}

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -22,6 +22,8 @@ Please visit [cucumber/CONTRIBUTING.md](https://github.com/cucumber/cucumber/blo
 
 ### Fixed
 
+* Replace `@cucumber/cucumber` deep imports with equivalents from main entry point
+
 ## [1.0.0-alpha.1](https://github.com/jbpros/cucumber-pretty-formatter/compare/v1.0.0-alpha.0...v1.0.0-alpha.1)
 
 ### Fixed

--- a/src/index.ts
+++ b/src/index.ts
@@ -4,12 +4,6 @@ import {
   IFormatterOptions,
   formatterHelpers,
 } from '@cucumber/cucumber'
-import {
-  getGherkinExampleRuleMap,
-  getGherkinScenarioMap,
-  getGherkinStepMap,
-} from '@cucumber/cucumber/lib/formatter/helpers/gherkin_document_parser'
-import { getPickleStepMap } from '@cucumber/cucumber/lib/formatter/helpers/pickle_parser'
 import * as messages from '@cucumber/messages'
 import * as CliTable3 from 'cli-table3'
 import { cross, tick } from 'figures'
@@ -18,7 +12,13 @@ import dedent from 'ts-dedent'
 
 import { makeTheme, ThemeItem, ThemeStyles } from './theme'
 
-const { formatLocation } = formatterHelpers
+const { formatLocation, GherkinDocumentParser, PickleParser } = formatterHelpers
+const {
+  getGherkinExampleRuleMap,
+  getGherkinScenarioMap,
+  getGherkinStepMap,
+} = GherkinDocumentParser
+const { getPickleStepMap } = PickleParser
 
 const marks = {
   [Status.AMBIGUOUS]: cross,

--- a/src/index.ts
+++ b/src/index.ts
@@ -1,6 +1,9 @@
-import { Status, SummaryFormatter } from '@cucumber/cucumber'
-import { IFormatterOptions } from '@cucumber/cucumber/lib/formatter'
-import { formatLocation } from '@cucumber/cucumber/lib/formatter/helpers'
+import {
+  Status,
+  SummaryFormatter,
+  IFormatterOptions,
+  formatterHelpers,
+} from '@cucumber/cucumber'
 import {
   getGherkinExampleRuleMap,
   getGherkinScenarioMap,
@@ -14,6 +17,8 @@ import { EOL as n } from 'os'
 import dedent from 'ts-dedent'
 
 import { makeTheme, ThemeItem, ThemeStyles } from './theme'
+
+const { formatLocation } = formatterHelpers
 
 const marks = {
   [Status.AMBIGUOUS]: cross,

